src/test/java/eggy/storage/StorageTest.java

```diff
@@ -0,0 +1,77 @@
+package eggy.storage;
+
+import eggy.exception.LoadTasksException;
+import eggy.task.TaskList;
+import eggy.task.Todo;
+import eggy.exception.EggyException;
+import eggy.exception.InvalidTaskTypeException;
+import eggy.exception.SaveTasksException;
+import eggy.task.Event;
+import eggy.task.Task;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
I like the clear organisations of imports by packages and alphabetical order, making it easier to remember.